recipes/jupyterlite/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyterlite" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlite-{{ version }}.tar.gz
+  sha256: 6e9949bc51247368bea8a0136256694255ace315e6e057020d29b36fe92c3c38
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling >=1.6.3,<2
+    - pip
+  run:
+    - python >=3.8
+    - jupyterlite-core
```

Comment:
probably a version pin, either directly, or in `run_constrained` to avoid mismatches down the line